src/main.rs

```diff
@@ -717,10 +717,15 @@ impl TypeDarwin {
 
 //     def is_option(self, line):
 //         return line.startswith('.It Fl')
-
+#[test]
+fn test_TypeDarwin_is_option() {
+    assert!(!TypeDarwin::is_option("Not an Option"));
+    assert!(TypeDarwin::is_option(".It Fl Is an Option"));
+    assert!(!TypeDarwin::is_option(""));
+}
```

Comment:
Formatting nit. Could you add a blank line between the test function and the `impl`?